src/app.ts

```diff
@@ -105,6 +111,13 @@ fedify
     )
     .setCounter(followingCounter);
 
+fedify
```

Comment:
If we're using this - then I think we need to remove the custom inbox handler on line 220, as well as the backing code behind it

Reply:
👍 Didn't realise we already had an inbox handler 🤦‍♂️ I'll move the logic to the existing handler and create an issue to refactor the existing handler to utilise `setInboxDispatcher` in the future